### PR TITLE
7.0 res partner fiscal position

### PR DESCRIPTION
**Pull request by lfreeke**

### CODE REVIEW

**Comment by NL66278** [comment]
:+1:  LGTM

There is a project to intelligently map countries (and even states) to fiscal positions: https://github.com/OCA/account-fiscal-rule

Might also be looked into as an alternative.

**Comment by lfreeke** [comment]
Thanks NL66278. I will look into it.

File: res_partner_fiscal_position/model/res_country.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+#
+##############################################################################
+from osv import fields, osv
+
+
+class Country(osv.osv):
+    _inherit = 'res.country'
+    _columns = {
+        'property_account_position': fields.property
+        ('account.fiscal.position',
```
Comment (hbrunn) [inline res_partner_fiscal_position/model/res_country.py]: put the opening bracket after `property`

**Comment by hbrunn** [comment]
@lfreeke is @NL66278's proposal the better alternative or do we still want to have this?

**Comment by lfreeke** [comment]
@hbrunn it is not a better alternative. We still need this.

**Comment by hbrunn** [comment]
@lfreeke okay, can you then fix my small comment, then I can merge it right away?

**Comment by lfreeke** [comment]
@hbrunn I will do

**Comment by NL66278** [comment]
@hbrunn and @lfreeke Small comment fixed, added .gitignore and some more style improvements.

**Comment by hbrunn** [comment]
looks good to me :+1:

**Pull request merged**